lib/redispub/publisher.go

```diff
@@ -159,6 +168,7 @@ func publishSingleMessage(p *Publication, client redis.UniversalClient, prefix s
 		strings.Join(p.Channels, "$"), // ARGV[3], channels
 	).Result()
 
+	metricLastCommandDuration.Set(time.Since(start).Seconds())
```

Comment:
Is this the most natural unit and not, say, `.Milliseconds()`?

Reply:
This is what a core prometheus developer [thinks](https://www.robustperception.io/who-wants-seconds/) as well. I'm generally not a fan but you're right, it is the de facto approach